app/models/miq_schedule/import_export.rb

```diff
@@ -64,7 +64,7 @@ def import_from_hash(miq_schedule, _options = nil)
         miq_schedule['userid'] = User.find_by(:id => report["userid"])
       end
 
-      miq_schedule["name"] = miq_schedule["name"] + "EUEUE"
```

Comment:
It looks like the whole line can be removed now, right?

Reply:
oh, sure, it is leftover, thanks for finding it 👍